Buổi 12/homework.js

```diff
@@ -0,0 +1,56 @@
+// Exercise 1
+const equalArray = (arr_1 = [], arr_2 = []) => {
+  if(arr_1 === arr_2) return true
+  if(arr_1.length !== arr_2.length) return false
+  for(let i=0; i<arr_1.length; i++) {
+    if(arr_1[i] !== arr_2[i]) {
+      return false
+    }
+  }
+  return true
+}
+
+// Exercise 2
+const flatArray = (arr = []) => {
+  return arr.flat(2)
+}
+
+// Exercise 3
+const chunkArray = (arr = [], number) => {
+  const new_arr = []
+  for(let i=0; i<arr.length; i+=number) {
+    new_arr.push(arr.slice(i, number+i))
+  }
+  return new_arr

```

Comment:
```suggestion
  return newArr
```
Tên biến phải đặt theo camel